src/util/logging.cpp

```diff
@@ -140,7 +146,8 @@ inline QString formatLogFileMessage(
         break;
     }
 
-    return levelName + QStringLiteral(" [") + threadName + QStringLiteral("] ") + message;
+    return timestamp + QStringLiteral(" ") + levelName + QStringLiteral(" [") +
+            threadName + QStringLiteral("] ") + message;
```

Comment:
Pattern are easier to read and seem to generate better code. https://compiler-explorer.com/z/eh15j6458

```suggestion
    return QStringLiteral("%1 %2 [%3] %4").arg(timestamp, levelname, threadName, message);
```